baselines/dev/setup-defaults.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+if [ ! -d $HOME/.pyenv ]
+then
+    # Install pyenv with the virtualenv plugin
+    curl https://pyenv.run | bash 
+
+    # To add the config to the right file (depends on the shell used)
+    rcfile=$HOME/.$(basename $SHELL)rc
+
+    # Add pyenv to $PATH
+    echo 'export PATH="$HOME/.pyenv/bin:$PATH"' >> $rcfile
+
+    # Init pyenv with the shell
+    echo 'eval "$(pyenv init -)"' >> $rcfile
+    echo 'eval "$(pyenv virtualenv-init -)"' >> $rcfile
+    source $rcfile
+fi
```

Comment:
Should we add an `else` block that updates `pyenv` (and the virtualenv plugin) by doing a `git pull`?

Reply:
Yes good idea ! I actually used the `pyenv update` built-in command.